### ISSUE TITLE
/multimon Not resizing correctly

### ISSUE THREAD

**Issue by TheOpenDevProject** [FreeRDP/FreeRDP#3360]
When I enter Multimon mode the initial connection is perfect but when I press `<ctrl>+<Alt>+<Enter>` and then again try to re-enter full screen mode during the same session the screen fails to resize and stays miniature.

![image](https://cloud.githubusercontent.com/assets/6072108/15489694/663de7d4-21a5-11e6-85e9-c8f08e017ec2.png)

Sorry for the blackout I was not able to show the contents of my desktop for sensitivity reasons.

Host O/S: Ubuntu 15.10 64bit

**Comment by bmiklautz** [comment]
@TheOpenDevProject you are using the package thac comes with Ubuntu 15.10?
If so could you give our nightlpy packagees a try and see if this fixes your issue? https://ci.freerdp.com/job/freerdp-nightly-binaries/

**Comment by TheOpenDevProject** [comment]
@bmiklautz Will do tomorrow. :+1:

**Comment by mclark1129** [comment]
I just installed the latest nightly on Ubuntu 16.04 and this issue was resolved for me.  I experienced the same behavior installing the xfreerdp package.

**Comment by begroff** [comment]
Using Ubuntu Xenial (16.04 LTS amd64) I noticed the same behavior when using `Control + Alt + Enter` to toggle during a multimon session with 3 monitors while using version **1.1.0-beta1**. It would successfully toggle out of full screen mode and when I did the keyboard combination to toggle back, it put all three monitors together very small and displayed it only on one of my monitors (it was all the way at the top of the screen). Plus it was slow to toggle back and forth and it had a hard time responding to the key combination.

I then installed the nightly version **2.0.0-dev4** and it worked like a charm. Doing the key combination of `Control + Alt + Enter `took it out of full screen mode and doing it again toggled it back to full screen mode on all of my three monitors! Plus, the responsiveness to toggling was super fast.
